README.md

```diff
@@ -1,8 +1,20 @@
 # multihash
 
+> hashpipe - pipe iff the hash matches
+
 Multihash is a protocol for differentiating outputs from various well-established cryptographic hash functions, addressing size + encoding considerations.
 
-It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See https://github.com/jbenet/random-ideas/issues/1 for a longer discussion.
+It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See [this GitHub thread](https://github.com/jbenet/random-ideas/issues/1) for a longer discussion.
```

Comment:
I disagree; I find parsing a URL in a paragraph of text to be difficult, and I don't think that we should ask the reader to do that. Switching it to "See jbenet/random-ideas#1" with a link would be a good compromise, I think, because that's more readable.